src/main/java/Bank.java

```diff
@@ -0,0 +1,60 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
+public class Bank {
+    private boolean success;
+    private String timestamp;
+    private String base;
+    private String date;
+    private Map<String, Double> rates;
```

Comment:
```suggestion
    private Map<String, BigDecimal> rates;
```

Reply:
The BigDecimal class offers better precision than Double

Reply:
Done